assets/stylesheets/pages/_user.scss

```diff
@@ -0,0 +1,5 @@
+.user-controller {
```

Comment:
Why using a directory named as `pages` instead of `screens` as per our conventions and as done by Tailwind as well 🤔